gcm_filters/kernels.py

```diff
@@ -290,6 +299,115 @@ def __call__(self, field: ArrayType):
 ALL_KERNELS[GridType.TRIPOLAR_POP_WITH_LAND] = POPTripolarLaplacianTpoint
 
 
+@dataclass
+class VectorLaplacian(BaseLaplacian):
```

Comment:
```suggestion
class CgridVectorLaplacian(BaseLaplacian):
```

Maybe this name would be a bit more specific?

I also wonder whether we should use different base classes, e.g. `BaseScalarLaplacian`, `BaseVectorLaplacian`. This would clarify a few ambiguities in the code.